Vite plugin peer dependency rejects Vite 7 and 8

The vite-plugin package.json declares peerDependencies vite ^5.0.0 || ^6.0.0, but Vite 7 and 8 are already released. Users on Vite 7+ get ERESOLVE errors when installing @floeorg/vite-plugin. Need to widen the peer dep range.

--- a/integrations/vite-plugin-floe/package.json
+++ b/integrations/vite-plugin-floe/package.json
@@ -28,7 +28,7 @@
   },
   "homepage": "https://github.com/milkyskies/floe",
   "peerDependencies": {
-    "vite": "^5.0.0 || ^6.0.0"
+    "vite": "^5.0.0 || ^6.0.0 || ^7.0.0 || ^8.0.0"
   },
   "devDependencies": {
     "@types/node": "^25.5.0",

